scripts/main.sh

```diff
@@ -15,6 +15,8 @@ declare -r TMP_TOKEN_FILE="$HOME/.op_tmux_token_tmp"
 declare -r OPT_SUBDOMAIN="$(get_tmux_option "@1password-subdomain" "my")"
 declare -r OPT_VAULT="$(get_tmux_option "@1password-vault" "")"
 declare -r OPT_COPY_TO_CLIPBOARD="$(get_tmux_option "@1password-copy-to-clipboard" "off")"
+declare -r OPT_ENABLED_URL_FILTER="$(get_tmux_option "@1password-enabled-url-filter" "on")"
+declare -r OPT_ITEMS_JQ_FILTER="$(get_tmux_option "@1password-items-jq-filter" "")"
```

Comment:
Since both options are related to the `jq filter`, why don't use a single option called `OPT_ITEMS_JQ_FILTER` where you can set the filter yourself (like you did it) and forget about the `OPT_ENABLED_URL_FILTER`?

In my opinion we should have only the option to allow the user to set her/his own filter, and if the user don't do this, we can get a default files (like the filter on the master branch)

Reply:
If you read above in the PR, this was per @yardnsm's recommendation. It's a much simpler user experience to just be able to set on and off the filtering.